src/components/common/InvitationBlock/InvitationBlock.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import { Heading4 } from '~shared/Heading';
+import Icon from '~shared/Icon';
+import Button from '~v5/shared/Button';
+import { useCopyToClipboard } from '~hooks/useCopyToClipboard';
+
+const displayName = 'common.InvitationBlock';
+
+const MSG = defineMessages({
+  buttonText: {
+    id: `${displayName}.buttonText`,
+    defaultMessage: `{isCopied, select,
+      true {Link copied}
+      other {Copy link}
+    }`,
+  },
+  inviteBlockTitle: {
+    id: `${displayName}.inviteBlockTitle`,
+    defaultMessage: 'Invite 1 person to create a Colony',
+  },
+  inviteBlockDescription: {
+    id: `${displayName}.inviteBlockDescription`,
+    defaultMessage:
+      'You can invite only one member to create a colony of their own using the new app during the private beta with this custom invite link: app.colony.io/createcolony/{invitationCode}',
+  },
+});
+
+// @TODO: Replace with real invitation code
```

Comment:
You can now query whether the user has an invite code attached to their user model after this commit https://github.com/JoinColony/colonyCDapp/pull/1197/commits/fdcbfac3511854ee2d3ecf02e63f9822ccabec9f